### PR TITLE
Add community pages

### PR DESCRIPTION
**Pull request by ysims**
Adds pages for Community chapter:

- Sponsors
- Collaborate

### CODE REVIEW

File: src/book/01-team/02-community/01-sponsors.mdx [hunk 1]
```diff
@@ -5,3 +5,15 @@ title: Sponsors
...
+
+![](.../.../.../images/UoN.png)
+
+![](.../.../.../images/4Tel.png)
+
+![](.../.../.../images/Tribotix.png)
+
+![](.../.../.../images/Elby.png)
+
+![](.../.../.../images/EMONA.png)
```
Comment (JosephusPaye) [inline src/book/01-team/02-community/01-sponsors.mdx]: We could just normalise all file names to lowercase:

```md
![](.../.../.../images/emona.png)
```

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 2]
```diff
@@ -8,7 +8,6 @@ slug: /projects/hardware/overview
 
 The platform used is the NUgus, based off the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling. Because of this, the components of the robot were split to allow for the size of the printer bed and the printer volume. The torso was also modified to have open shoulder covers for shoulder servo cable access.
 
-![Coronal image of the NUgus robot](../images/front-robot.png)
-![Sagittal image of the NUgus robot](../images/side-robot.png)
+![Coronal and saggital images of the NUgus robot](../images/NUgus.png)
```
Comment (JosephusPaye) [inline src/book/02-projects/03-hardware/01-overview.mdx]: Maybe remove this change and re-add when you do the hardware pages.

File: src/book/01-team/02-community/01-sponsors.mdx (hunk 1 above)
Comment (ysims) [reply src/book/01-team/02-community/01-sponsors.mdx]: Done :+1:

File: src/book/02-projects/03-hardware/01-overview.mdx (hunk 2 above)
Comment (ysims) [reply src/book/02-projects/03-hardware/01-overview.mdx]: Done :+1:

**Pull request merged**